content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: influxctl auth
+description: >
+  The `influxctl auth` command and its subcommands provide the ability to
+  login and logout.
+menu:
+  influxdb_clustered:
+    parent: influxctl
+weight: 201
+---
+
+The `influxctl auth` command and its subcommands provide information about
+{{< product-name >}} clusters.
+
+## Usage
+
+```sh
+influxctl auth [subcommand] [subcommand options] [arguments...]
+```
+
+## Subcommands
+
+| Subcommand                                                               | Description                     |
+| :----------------------------------------------------------------------- | :------------------------------ |
+| [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Login with InfluxData Auth0 or InfluxDB Clustered identity provider |
```

Comment:
> InfluxDB Clustered identity provider

Does this only apply to Clustered, or to Cloud Dedicated as well?

Reply:
Both, dedicated is the InfluxData Auth0

Reply:
Right